wx_mpl_dynamic_graph.py

```diff
@@ -1,127 +1,132 @@
 """
 GP:
-Changed datasource, title, and refresh interval to use
-as a poor man's Arduino oscilliscope.
+Changed datasource, title, and refresh interval to use as a poor man's Arduino
+oscilliscope.
 
-This demo demonstrates how to draw a dynamic mpl (matplotlib) 
-plot in a wxPython application.
+This demo demonstrates how to draw a dynamic matplotlib plot in a wxPython
+application.
 
-It allows "live" plotting as well as manual zooming to specific
-regions.
+It allows "live" plotting as well as manual zooming to specific regions.
 
-Both X and Y axes allow "auto" or "manual" settings. For Y, auto
-mode sets the scaling of the graph to see all the data points.
-For X, auto mode makes the graph "follow" the data. Set it X min
-to manual 0 to always see the whole data from the beginning.
+Both X and Y axes allow "auto" or "manual" settings. For Y, auto mode sets the
+scaling of the graph to see all the data points. For X, auto mode makes the
+graph "follow" the data. Set it X min to manual 0 to always see the whole data
+from the beginning.
 
-Note: press Enter in the 'manual' text box to make a new value 
-affect the plot.
+Note: press Enter in the 'manual' text box to make a new value affect the plot.
 
 Eli Bendersky (eliben@gmail.com)
 License: this code is in the public domain
 Last modified: 31.07.2008
 """
 import os
-import pprint
-import random
-import sys
 import wx
+import numpy as np
 
-REFRESH_INTERVAL_MS = 90
-
-# The recommended way to use wx with mpl is with the WXAgg
-# backend. 
-#
 import matplotlib
-matplotlib.use('WXAgg')
 from matplotlib.figure import Figure
-from matplotlib.backends.backend_wxagg import \
-    FigureCanvasWxAgg as FigCanvas, \
-    NavigationToolbar2WxAgg as NavigationToolbar
-import numpy as np
-import pylab
-#Data comes from here
+
 from Arduino_Monitor import SerialData as DataGen
 
+# The recommended way to use wx with mpl is with the WXAgg backend.
+matplotlib.use('WXAgg')
+
+# Those import have to be after setting matplotlib backend.
```

Comment:
Go ahead and make the changes!  Looks great.
Going forward, as long as you've verified the changes using an actual Arduino (or are otherwise sure they'll work) feel free to change anything you want.  No need to wait on me.